react/react2/week1/github_user/src/Components/APIOutput.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext, useEffect, useState } from "react";
+import UserList from "./UserList";
+import Blocker from "./Blocker";
+import { SearchInputContext } from "./Contexts";
+export default function APIOutput() {
+  const [data, setData] = useState();
+  const [isLoaded, setIsLoaded] = useState();
+  const [errorData, setErrorData] = useState();
+  const inputText = useContext(SearchInputContext);
+  const loadData = async () => {
+    let res = [];
+    setErrorData();
+    setData();
+    setIsLoaded();
```

Comment:
setErrorData, setData, etc. functions are called when you want to change the previous value of that variable.